runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java

```diff
@@ -44,10 +45,18 @@
      */
     final CorfuRuntime runtime;
 
+    /**
+     * Name of the wrapped stream.
+     */
+    @Getter
+    private final String streamName;
```

Comment:
This is not needed. Since you have access to the underlaying stream (i.e. passed through the constructor), then you can get the name via `streamView.getName()`

Reply:
See above